### PR TITLE
Refactor: Web client useless code removed

### PR DESCRIPTION
**Pull request by georgolden**
- [x] tests and linter show no problems (`npm t`)
- [x] code is properly formatted (`npm run fmt`)

### CODE REVIEW

File: dist/metacom.js [hunk 1]
```diff
@@ -233,16 +221,11 @@ class HttpTransport extends Metacom {
...
-          this.message(packet);
-        });
-      }
-      throw new Error(`Status Code: ${status}`);
-    });
+    }).then((res) =>
+      res.text().then((packet) => {
+        this.message(packet);
+      })
+    );
```
Comment (tshemsedinov) [inline dist/metacom.js:228]: I am not sure we can simplify this

File: dist/metacom.js [hunk 2]
```diff
@@ -19,21 +19,9 @@ class MetacomError extends Error {
   }
 }
 
-class MetacomInterface {
+class MetacomInterface extends EventEmitter {
```
Comment (tshemsedinov) [inline dist/metacom.js:22]: :+1:

**Review by tshemsedinov — COMMENTED** [review]
Need a call to land this

**Comment by georgolden** [comment]
@tshemsedinov please land

**Comment by georgolden** [comment]
@tshemsedinov improved commit names. Thanks for a video review. Now can be merged without any changes.

**Review by tshemsedinov — APPROVED** [review]

**Comment by tshemsedinov** [comment]
Landed in https://github.com/metarhia/metacom/commit/6553cf6dace8a18994e80754559f312d50af4b2a and https://github.com/metarhia/metacom/commit/9d74d8d0caa4b07380d401044b967ff7863dbe5a

**Pull request closed without merging**